app/cli/cmd/workflow_workflow_run_describe.go

```diff
@@ -78,6 +89,9 @@ func newWorkflowWorkflowRunDescribeCmd() *cobra.Command {
 		publicKey = os.Getenv(signingKeyEnvVarName)
 	}
 
+	cmd.Flags().StringVar(&certPath, "cert", "", "public certificate in PEM format to be used to verify the attestation")
+	cmd.Flags().StringVar(&chainPath, "chain", "", "certificate chain (intermediates, root) in PEM format to be used to verify the attestation")
```

Comment:
since the number of options is growing, maybe I'd stick to these flags same than `cosign verify`, `cert-chain` instead of `chain`. WDYT?

```
  cosign verify --cert cosign.crt --cert-chain chain.crt <IMAGE>
```